### ISSUE TITLE
Alt text doesn't read with JAWS screen reader

### ISSUE THREAD

**Issue by kirsty-hames** [cgkineo/adapt-graphicLottie#28]
### Subject of the issue
When navigating a course with JAWS screen reader, the animation is skipped and the alt text is not read.

Alt text reads fine with NVDA and voiceOver.

### Steps to reproduce
- Set `_graphic.alt` text for animation.
- With JAWS running, navigate to the animation and note this is skipped in the keyboard order and the alt text isn't read.
